<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViPlaybackWidget</class>
 <widget class="QWidget" name="ViPlaybackWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>89</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>950</width>
    <height>1050</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="ViPushButton" name="recordButton" native="true"/>
   </item>
   <item row="0" column="1">
    <widget class="ViPushButton" name="playButton" native="true"/>
   </item>
   <item row="0" column="2">
    <widget class="ViPushButton" name="pauseButton" native="true"/>
   </item>
   <item row="0" column="3">
    <widget class="ViPushButton" name="stopButton" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ViPushButton</class>
   <extends>QWidget</extends>
   <header>vipushbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
